contents/blog/pricing-lessons.md

```diff
@@ -0,0 +1,104 @@
+
+* We differentiated on features
+* Our usage grows with our customers' products. If we charged a flat fee, we couldn't do this so easily.
+
+Just because it's easy to get started, doesn't mean your product has to be cheap.
+
+## You don't have to do all these things immediately
+
+Early on, optimize for learning. Later on, optimize for the best solution for your users.
+
+**We didn't start with a $0 minimum**. Last summer, we worked on finding just five reference customers for our paid product (we now have X customers).
+
+For these, we didn't offer $0 minimum - we charged $2k/month minimum. We simply couldn't support more than this, and we were already getting enough inbound interest.
+
+**We didn't start with self-serve sales**, as we wanted to talk to users at first. This meant we grew more slowly to optimize for learning at the start. Once we had a reasonable formula, we could accelerate things by chopping out any manual sales process.
+
+## Pricing to value is dumb
+
+Conventional wisdom is to "price to value", which is broadly interpreted as you should charge a given customer the most you can.
+
+This gives you no room - if the service is at all below initial expectations, the customer may churn. And given you likely don't know the precise dollar value for a given customer, you'll end up losing customers for pricing reasons, or at least being a pain to negotiate with. These are two good ways to get less growth. 
+
+There are exceptions to this rule - relationship-driven enterprise sales, for example. But, long term, anyone pricing like this is asking to get disrupted.
+
+## Make pricing work for your biggest customers
+
+We implemented logarithmically-decling pricing with usage. This means something like - every time your volume goes up 10x, the incremental price decrease 5x.
+
+## Don't try to make money off people that don't have money
+
+This was one of our biggest lessons. We started by focussing on startups but realized our lowest churn teams were working on products that already had product market fit.
+
+Trying to get money out of tiny startups would have been a mistake.
+
+## We're not done yet
+
+If you like what you're reading, you'll probably have had a peek at [PostHog's pricing](../pricing).
+
+Before you copy paste, we've room for improvement. Here are a few things we're working on next.
+
+* Transparent enterprise pricing. We're currently figuring out how to standardize this by talking to users.
+* More clarity on about what you get if you upgrade.
+* An enterprise cloud offering
+* Some way of capturing market share for _extremely high volume_ but _low marginal value_ users
+
+## The results
+
+TODO: insert chart just showing shape of revenue growth
+
+Enough said.
+
+## Pricing as a product
+
+Think of your pricing like a product - you want users to become paying customers. What problems do users have from where they are today, to becoming paying customers tomorrow?
+
+Solve those.
```

Comment:
I feel like we should end this piece on the 'The results' rather than having that extra point at the end. Any way we could have 'Pricing as a product' at the start? Maybe we put that intro and the piece is about how we used that approach to "nail pricing"

Reply:
Added